hackIT/templates/index.html.tera

```diff
@@ -1,11 +1,15 @@
 <!DOCTYPE html>
 <html>
   <head>
+  <link rel="stylesheet" type="text/css" href="/static/styles.css">
     <title>CODE<>IT</title>
   </head>
   <body>
-    <h1>Hello {{name}}</h1>
-    <a href={{auth_url}}>login</a>
-    <h3>{{auth}} </h3>
+    <div class="wrapper">
+      <h1>Welcome to hack_IT!</h1>
+      <p>to acces the challenges, please log in on gamma!</p>
+      <a href="www.google.se"> [=LOGIN=] </a>
```

Comment:
replace www.google.se with `{{auth}}`. Then gamma will work out of the box

Reply:
Did this and it now the link redirects to ./you and says "request not found" or alternatively ./you are not logged in...
not sure of how or why this is happening atm

Reply:
Opps meant auth_url, but we can probably leave it since its outside of the scope of the pr